docs/Основные блоки и устройство компьютера/Основные внешние порты.md

```diff
@@ -0,0 +1,137 @@
+оснащены S / PDIF через TOSLINK.
+- TOSLINK (Toshiba Link) - это наиболее часто используемый цифровой аудиопорт,
+который может поддерживать 7.1-канальный объемный звук с помощью всего одного кабеля.
+ 
+## **Видеопорты**
+
+### Порт VGA
+
+Порт VGA используется во многих компьютерах, проекторах, видеокартах
+и телевизорах высокой четкости. Это разъем D-sub, состоящий из 15 контактов
+в 3 ряда. Порт VGA - это основной интерфейс между компьютерами и более старыми ЭЛТ-мониторами. 
+Даже современные ЖК-мониторы и светодиодные мониторы поддерживают порты VGA, но качество изображения ухудшается.
+VGA передает аналоговые видеосигналы с разрешением до 648X480.
+
+### Цифровой видеоинтерфейс (DVI)
+
+DVI - это высокоскоростной цифровой интерфейс между контроллером дисплея,
+таким как компьютер, и устройством отображения, таким как монитор. Он был разработан
+с целью передачи цифровых видеосигналов без потерь и замены аналоговой технологии VGA.
+Существует три типа разъемов DVI в зависимости от передаваемых сигналов:
+DVI-I, DVI-D и DVI-A. 
+- DVI-I - это порт DVI со встроенными аналоговыми и цифровыми сигналами. 
+- DVI-D поддерживает только цифровые сигналы, 
+- DVI-A поддерживает только аналоговые сигналы. 
+Цифровые сигналы могут быть как одинарными, так и двойными, где одиночный канал поддерживает цифровой сигнал с разрешением до 1920X1080, а двойной канал поддерживает цифровой сигнал с разрешением до 2560X1600. 
+
+### Display Por
+
+Display Port (DP) - это интерфейс цифрового дисплея с дополнительным многоканальным звуком и другими формами данных.
+Display Port разработан с целью замены портов VGA и DVI в качестве основного интерфейса между компьютером и монитором.
+Последняя версия DisplayPort 1.3 поддерживает разрешение до 7680 X 4320.
+
+### Разъем RCA
+
+Разъем RCA может передавать композитные видео- и стереофонические аудиосигналы по трем кабелям.
+Композитное видео передает аналоговые видеосигналы, а разъем выполнен в виде разъема RCA желтого цвета.
+Видеосигналы передаются по одному каналу вместе с импульсами строчной и кадровой синхронизации
+с максимальным разрешением 576i (стандартное разрешение).
+ Красный и белый разъемы используются для стереофонических аудиосигналов
+(красный для правого канала и белый для левого канала).
+
+### HDMI
+
+HDMI - это аббревиатура от High Definition Media Interface. HDMI - это цифровой интерфейс
+для подключения устройств высокого и сверхвысокого разрешения, таких как компьютерные мониторы,
+телевизоры высокой четкости, проигрыватели Blu-Ray, игровые консоли, камеры высокого разрешения и т. Д.
+HDMI можно использовать для передачи несжатого видео и сжатых или несжатых аудиосигналов. 
+Разъем HDMI состоит из 19 контактов и последней версии HDMI, т.е. 
+HDMI 2.0 может передавать цифровой видеосигнал с разрешением до 4096 × 2160 и 32 аудиоканала
+
+
+## **Порты подключения переферии**
+
+###  PS/2
+Разъем PS / 2 разработан IBM для подключения мыши и клавиатуры. Он был представлен
```

Comment:
[ch-md-error-separate-blocks](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-separate-blocks)